extensions/amp-ad/0.1/legacy-ad-intersection-observer-host.js

```diff
@@ -221,10 +221,13 @@ export class LegacyAdIntersectionObserverHost {
       this.intersectionObserver_.observe(this.baseElement_.element);
     }
     if (!this.fireInOb_) {
-      this.fireInOb_ = new IntersectionObserver((entries) => {
-        const lastEntry = entries[entries.length - 1];
-        this.sendElementIntersection_(lastEntry);
-      });
+      this.fireInOb_ = new IntersectionObserver(
+        (entries) => {
+          const lastEntry = entries[entries.length - 1];
+          this.sendElementIntersection_(lastEntry);
+        },
+        {root: this.baseElement_.win.document}
```

Comment:
Is this going to behave correctly? We've changed the root from the overall viewport to the document viewport.

Reply:
In all Viewer cases besides for emails the viewer is 100% height/width. email doesn't support amp-ad so its a nonissue in this case